<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogMapProperties</class>
 <widget class="QDialog" name="DialogMapProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Map properties...</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_11">
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,0,0,0,1">
         <item>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="0">
            <widget class="QLabel" name="labelName">
             <property name="text">
              <string>Name:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="WidgetTextLang" name="widgetLangName" native="true">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="2" rowspan="2">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>68</width>
               <height>58</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="labelTileset">
             <property name="text">
              <string>Tileset:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="comboBoxTileset">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBoxSizes">
           <property name="title">
            <string>Sizes</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_3">
              <property name="horizontalSpacing">
               <number>25</number>
              </property>
              <property name="verticalSpacing">
               <number>15</number>
              </property>
              <item row="0" column="0">
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QLabel" name="labelLength">
                  <property name="text">
                   <string>Length:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinBoxLength">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>75</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>75</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>1000</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="0" column="1">
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QLabel" name="labelWidth">
                  <property name="text">
                   <string>Width:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinBoxWidth">
                  <property name="minimumSize">
                   <size>
                    <width>75</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>75</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>1000</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="0" column="2" rowspan="2">
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>78</width>
                  <height>98</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="1" column="0">
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QLabel" name="labelheight">
                  <property name="text">
                   <string>Height:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinBoxHeight">
                  <property name="minimumSize">
                   <size>
                    <width>75</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>75</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>1000</number>
                  </property>
                  <property name="value">
                   <number>1</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="1" column="1">
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <widget class="QLabel" name="labelDepth">
                  <property name="text">
                   <string>Depth:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinBoxDepth">
                  <property name="minimumSize">
                   <size>
                    <width>75</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>75</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="maximum">
                   <number>1000</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="1" column="0">
            <widget class="QLabel" name="labelBackgroundSound">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Background sound:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2" rowspan="3">
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>88</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="labelMusic">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Music:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="WidgetSong" name="widgetChooseMusic" native="true">
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>27</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="labelCameraPropID">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Camera prop. ID:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="WidgetSong" name="widgetChooseBackgroundSound" native="true">
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>27</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="PanelPrimitiveValue" name="panelPrimitiveCameraProperties" native="true">
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBoxSky">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="title">
            <string>Sky</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="0" column="0">
               <widget class="QRadioButton" name="radioButtonColor">
                <property name="text">
                 <string>Color ID:</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="PanelPrimitiveValue" name="panelPrimitiveValueSkyColor" native="true">
                <property name="minimumSize">
                 <size>
                  <width>250</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>250</width>
                  <height>27</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="0" column="2" rowspan="3">
               <spacer name="horizontalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>98</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="1" column="0">
               <widget class="QRadioButton" name="radioButtonPicture">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Picture:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="WidgetPicture" name="widgetPictureSky" native="true">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>250</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>250</width>
                  <height>27</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QRadioButton" name="radioButtonSkybox">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Skybox ID:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="PanelPrimitiveValue" name="panelPrimitiveValueSkyBoxID" native="true">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>250</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>250</width>
                  <height>27</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Map startup reactions</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_12">
           <item row="0" column="0">
            <widget class="QScrollArea" name="scrollArea">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>610</width>
                <height>457</height>
               </rect>
              </property>
              <layout class="QGridLayout" name="gridLayout_9">
               <item row="0" column="0">
                <widget class="PanelObject" name="panelObject" native="true"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Battles</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_10">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_7" stretch="0,1">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1">
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Battle map:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboBoxBattleMap">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>200</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>200</width>
                   <height>16777215</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_2">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="title">
                <string>Random battles</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="0" column="0">
                 <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
                  <item>
                   <widget class="QTreeView" name="treeViewRandomBattles"/>
                  </item>
                  <item>
                   <layout class="QGridLayout" name="gridLayout">
                    <item row="0" column="0">
                     <widget class="QLabel" name="label_11">
                      <property name="text">
                       <string>Number of steps after next battle:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="1" rowspan="4">
                     <spacer name="horizontalSpacer_4">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>18</width>
                        <height>118</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item row="1" column="0">
                     <widget class="PanelPrimitiveValue" name="widgetConstantVariableSteps" native="true">
                      <property name="minimumSize">
                       <size>
                        <width>250</width>
                        <height>27</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>250</width>
                        <height>27</height>
                       </size>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="0">
                     <widget class="QLabel" name="label_12">
                      <property name="text">
                       <string>with variation (in %):</string>
                      </property>
                     </widget>
                    </item>
                    <item row="3" column="0">
                     <widget class="PanelPrimitiveValue" name="widgetConstantVariableStepsVariation" native="true">
                      <property name="minimumSize">
                       <size>
                        <width>250</width>
                        <height>27</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>250</width>
                        <height>27</height>
                       </size>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PanelPrimitiveValue</class>
   <extends>QWidget</extends>
   <header>panelprimitivevalue.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetTextLang</class>
   <extends>QWidget</extends>
   <header>widgettextlang.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetSong</class>
   <extends>QWidget</extends>
   <header>widgetsong.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PanelObject</class>
   <extends>QWidget</extends>
   <header>panelobject.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetPicture</class>
   <extends>QWidget</extends>
   <header>widgetpicture.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogMapProperties</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogMapProperties</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
